pkg/sql/crdb_internal.go

```diff
@@ -8929,3 +8930,26 @@ func populateFlowTokensResponse(
 	}
 	return nil
 }
+
+var crdbInternalClusterReplicationResolvedView = virtualSchemaView{
+	schema: `
+		CREATE VIEW crdb_internal.cluster_replication_spans AS WITH spans AS (
+			SELECT
+				job_id, jsonb_array_elements(crdb_internal.pb_to_json('progress', value)->'streamIngest'->'checkpoint'->'resolvedSpans') AS s
+			FROM system.job_info
+			WHERE info_key = 'legacy_progress' AND crdb_internal.pb_to_json('progress', value)->'streamIngest' IS NOT NULL
```

Comment:
Interesting. I suppose on the one hand it is nice for this query to only touch the job table. On the other hand I have to imagine that in some cases something like `WHERE job_id IN (SELECT id FROM system.jobs WHERE type = 'STREAM INGESTION'`) (or whatever the right type is) might be faster than than scanning every progress.

I leave that to your judgement. We can always change it later.